### ISSUE TITLE
focused app: none

### ISSUE THREAD

**Issue by Kyle-W-OBrien** [Martenfur/TabletFriend#34]
I am on a Surface Pro X (arm64) and Tablet Friend does not seem to register when an application has focus, which means I can't use any app specific layouts. This isn't a critical issue, but it does diminish the utility a little bit. This occurs with both native applications like Edge, and other applications, and regardless of whether the toolbar is docked or floating.

**Comment by Martenfur** [comment]
Honestly, I'm surprised it even launches on ARM. But yeah, I only support x86 at the moment, so you're on your own with this one, I can't even test it. You can try compiling it for ARM manually, maybe that would work I dunno.

**Comment by Kyle-W-OBrien** [comment]
Hey it's a great little tool - even on arm! The app focus is the only issue I have encountered so far. I went ahead and compiled it for arm64 and the app focus seems to have been resolved as a result. I've attached the zipped arm release if you want to add it to the downloads page for the app.

Thanks again for the great work!

[Tablet Friend - arm64.zip](https://github.com/Martenfur/TabletFriend/files/13275390/Tablet.Friend.-.arm64.zip)
